querries.sql

```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
```

Comment:
- [ ] **There is a typo error 😨** Kindly rename this file to `queries.sql` instead of `querries.sql` to make it more professional 👍

Reply:
@TuyishimireEric, I wanted to reach out to you and let you know that I have made all the changes you requested me to make. I wanted to express my gratitude for your feedback and guidance throughout this process. I am always happy to collaborate and make any further adjustments that may be required.

Thank you again for your time and support.